docs/enhancements/api-for-skipping-validations.md

```diff
@@ -0,0 +1,144 @@
+
+## Proposal
+
+I think the API should be kept reasonably simple.
+
+### List existing validations.
+
+GET api/v2/skippedValidations
+Will return a JSON array of all the validations that are currently being skipped, format similar to
+
+```
+{
+  "host-skipped-validations": [
+    "no-ip-collisions-in-network",
+    "sufficient-network-latency-requirement-for-role'"
+  ],
+  "cluster-skipped-validations": [
+    "network-prefix-valid",
+    "api-vips-defined"
+  ]
+}
+```
+
+### Add some skipped validations.
+
+POST api/v2/skipped-validations
+
+Essentially a document will be submitted via POST in JSON
+```
+{
+  "skipped-host-validations": [
+    "no-ip-collisions-in-network"
+  ],
+  "skipped-cluster-validations": [
+    "api-vips-defined"
+  ]
+}
+```
+
+* Any entries found will be added to the existing list of validations to be skipped. This will appropriately de-deuplicated and no error will be returned if duplication is attempted.
+* If an attempt is made to reference a validation that does not exist then a warning will be logged but this will not be considered fatal.
+* After the call has completed, if this leads to a change in the collection of skipped validations then an event will be emitted.
+
+### Edit the validations
+
+PUT api/v2/skipped-validations
+
+Essentially a document will be submitted via POST in JSON
+```
+{
+  "skipped-host-validations": [
+    "no-ip-collisions-in-network"
+  ],
+  "skipped-cluster-validations": [
+    "api-vips-defined"
+  ]
+}
+```
+* The freshly submitted JSON document, will be the new source of truth. All existing "skip operations" will be undone before applying this.
+* If an attempt is made to reference a validation that does not exist then a warning will be logged but this will not be considered fatal.
+* Submitting an empty doc '{}' will lead to the deletion of all skipped validations.
+* After the call has completed, if this leads to a change in the collection of skipped validations then an event will be emitted.
+
+### Effect on validation.
+
+When a validation is skipped then this should be reflected in the Validation status. For example;
+
+```
+{
+    "id": "api-vips-defined",
+    "status": "skipped",
+    "message": "API virtual IPs validation has been suppressed by the user"
+}
+```
+
+### User Stories
+
+#### Story 1
+
+As a Kaloom user, I would like to be able to bypass VIP validation in cases where VIPS could be incorrectly shown as busy. So that I may proceed with installation.
+
+"Kaloom has very specific setup where vips can be shown as busy though installation can proceed with them.
+Currently they need to override vips in install config to be able to install cluster.
+After adding the new api they can just run it and skip this specific validation."
+
+### Risks and Mitigations
+
+- We should not add uneccesary noise to triage tickets with this, it is therefore quite important to add some signatures to indicate that these validations have been bypassed.
```

Comment:
We should skip those tickets same way we do for jupiter installations, just no need to see them at all, i think

Reply:
That would be nice, do you have any information on how that skipping is set up?

Reply:
https://github.com/openshift-assisted/assisted-installer-deployment/blob/463dc6612c923c46d47e80762c72651758a690d3/tools/create_triage_tickets.py#L64

Reply:
Thank you